libs/data-mapper/src/lib/utils/Connection.Utils.ts

```diff
@@ -250,49 +250,15 @@ export const isValidConnectionByType = (srcDataType: NormalizedDataType, tgtData
     return true;
   }
 
-  switch (tgtDataType) {
```

Comment:
Don't you want to keep this? We only need to convert when the types are incompatible. Like a decimal can be used in a number with no conversion necessary

Reply:
The backend team changed allowed types to explicitly mention "decimal" and "number" so we don't have to infer anything on the frontend. So for your example we would still get the same result.
Ex: 
"allowedTypes": [
                        "Number",
                        "Integer",
                        "Decimal"
                    ],